third-party/crowdsec/README.md

```diff
@@ -0,0 +1,67 @@
+# James' extensions for Crowdsec
+
+This module is for developing and delivering extensions to James for the [Crowdsec](https://www.crowdsec.net/) (IP filtering system)
+
+## How to run
+
+- The Crowdsec extension requires an extra configuration file `crowdsec.properties` to configure Crowdsec connection
+  Configuration parameters:
+    - `crowdsecUrl` : URL defining the Crowdsec's bouncer. Eg: http://crowdsec:8080/v1
+    - `apiKey` : Api key for pass authentication when request to Crowdsec's bouncer.
+
+- Declare the `extensions.properties` for this module.
+
+```
+guice.extension.module=org.apache.james.module.CrowdsecModule
+```
+
+- Declare the Crowdsec EhloHook in `smtpserver.xml`. Eg:
+
+```
+<handlerchain>
+    <handler class="org.apache.james.smtpserver.fastfail.ValidRcptHandler"/>
+    <handler class="org.apache.james.smtpserver.CoreCmdHandlerLoader"/>
+    <handler class="org.apache.james.CrowdsecEhloHook"/>
+</handlerchain>
+```
+
+- Docker compose file example: [docker-compose.yml](docker-compose.yml).
+- The sample-configuration: [sample-configuration](sample-configuration)
+- For running docker-compose, first compile this project
+
+```
+mvn clean install -DskipTests
+```
+then run it: `docker-compose up`
+
+## Additional webadmin endpoints
+
+### Check IP blocked by Crowdsec (decisions)
```

Comment:
This is misleading. These are CrowdSec endpoints, not (James) additional webadmin endpoints.

Maybe just put a link to CrowdSec endpoints documentation, or change the way you paraphrase.